package.json

```diff
@@ -85,6 +85,11 @@
     "typescript": "^3.3.3333",
     "semantic-release": "^15.13.3"
   },
+  "peerDependencies": {
```

Comment:
Hmm, what about non-typescript users. They shouldn't be forced to add these dependencies to their project.

Reply:
That was my thought too, that's why I didn't add them initially, but we've had a few users (see issue) complain about 'weird' errors due to these packages not being installed.

`peerDependencies` don't automatically install but give you a small warning which you can ignore, might be a fair trade off.

Reply:
@SimonSchick @sushantdhiman I am starting to think that `peerDependencies` is a bad idea too. I wouldn't like to see these as a non-TS user. What if we add a script to run once sequelize is loaded, to auto detect if typescript is being used, check if the dependencies are installed and print a warning message automatically on the console if they're missing?

Reply:
Also, about "some users complaining", I think that once the rest of this PR is merged (without peerDependencies), the documentation will be much clearer on TypeScript usage and these mistakes will be less frequent. How about just removing the `peerDependencies` thing from this PR and waiting a bit to see if the confusion remains?

Reply:
Someone else just had the same problem. I think this PR should be merged ASAP - but without the `peerDependencies` part. If the confusion persists, then we can think about it again.

(this is just my opinion, of course)

Reply:
@SimonSchick remove peer dependencies and merge it